### PR TITLE
Set gphoto2 faster

### PR DESCRIPTION
**Pull request by jamessynge**
Set multiple gphoto2 properties in a single command, and support setting by both index or value.
Fix regular expression literals that should have been raw-string-literals (i.e. needed r prefix).

### CODE REVIEW

File: pocs/camera/camera.py [hunk 1]
```diff
@@ -311,6 +311,42 @@ def set_property(self, prop, val):
...
+            set_cmd.extend(['--set-config-index', '{}={}'.format(prop, val)])
+        for prop, val in prop2value.items():
+            set_cmd.extend(['--set-config-value', '{}={}'.format(prop, val)])
+
+        self.command(set_cmd)
+
+        # Forces the command to wait
+        self.get_command_result()
+
+    def set_property_values(self, prop_vals):
```
Comment (wtgee) [inline pocs/camera/camera.py]: I think we can just drop this method if we have the above and the individual `set_property`

File: pocs/camera/canon_gphoto2.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import datetime
```
Comment (wtgee) [inline pocs/camera/canon_gphoto2.py]: Not terribly important here, but we try to make all date/time items come from `current_time` (which allows us to override time in a simulator mode). Could be argued that here you don't ever want to do that and we always want the current.

File: pocs/camera/canon_gphoto2.py [hunk 3]
```diff
@@ -33,25 +34,29 @@ def connect(self):
...
+            '/main/capturesettings/focusmode': 0,         # Manual (don't try to focus)
+            '/main/capturesettings/continuousaf': 0,      # No auto-focus
+            '/main/capturesettings/autoexposuremode': 3,  # 3 - Manual; 4 - Bulb
+            '/main/capturesettings/drivemode': 0,         # Single exposure
+            '/main/capturesettings/shutterspeed': 0,      # Bulb
+        }
+        prop2value = {
+            '/main/settings/artist': 'Project PANOPTES',
+            '/main/settings/ownername': 'Project PANOPTES',
+            '/main/settings/copyright': 'Project PANOPTES {}'.format(datetime.datetime.today().year),
```
Comment (wtgee) [inline pocs/camera/canon_gphoto2.py]: As per comment above, this can be:
```
current_time().datetime.year
```

File: pocs/camera/camera.py (hunk 1 above)
Comment (jamessynge) [reply pocs/camera/camera.py]: Done.

File: pocs/camera/canon_gphoto2.py (hunk 2 above)
Comment (jamessynge) [reply pocs/camera/canon_gphoto2.py]: Done.

File: pocs/camera/canon_gphoto2.py (hunk 3 above)
Comment (jamessynge) [reply pocs/camera/canon_gphoto2.py]: Done.

**Comment by jamessynge** [comment]
Fixed trailing whitespace.

**Pull request merged**